libbeat/docs/yaml.asciidoc

```diff
@@ -49,7 +49,7 @@ ifdef::allplatforms[]
 
 ["source","shell",subs="attributes"]
 ----------------------------------------------------------------------
-{beatname_lc} -c {beatname_lc}.yml -configtest
+{beatname_lc} test config -c {beatname_lc}.yml
 ----------------------------------------------------------------------
 
 endif::allplatforms[]
```

Comment:
The Windows section that follows looks like it needs the same fix.

Reply:
I've fixed the Windows section here: #7360